test/runtests.jl

```diff
@@ -2162,6 +2162,40 @@ end
     end
 end
 
+# issue #354
+@testset "entr with modules" begin
+
+    testdir = newtestdir()
+    modname = "A354"
+    srcfile = joinpath(testdir, modname * ".jl")
+
+    function setvalue(x)
+        open(srcfile, "w") do io
+            print(io, "module $modname test() = $x end")
+        end
+    end
+
+    setvalue(1)
```

Comment:
I think you want `sleep(mtimedelay)` before & after the `@eval using` for reasons we discussed in #354. I know these tests don't run on OSX by default but we might as well get them as close to passing as possible.

Reply:
Right. This one actually *does* run on macOS at the moment, as it seemed to work just fine on my machine – but I can add the same conditional around it as the other `entr` stuff, if you'd like. As for the sleep – I didn't think it applied at that point in the code? (I'm just importing the module for the first time?) But no problem, I'll add that.

Reply:
Maybe you were right, didn't notice the sleep after the `@async` loop. Still can't hurt.